pkgs/by-name/je/jellyfin-tui/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  mpv,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jellyfin-tui";
+  version = "v1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "dhonus";
+    repo = pname;
```

Do not reuse `pname` for the fetcher. Write the string literal explicitly.
```suggestion
    repo = "jellyfin-tui;
```